test/e2e/specs/editor/editor__post-advanced-flow.ts

```diff
@@ -58,7 +58,8 @@ describe( DataHelper.createSuiteTitle( `Editor: Advanced Post Flow` ), function
 		it( 'Enter post content', async function () {
 			const blockHandle = await editorPage.addBlockFromSidebar(
 				ParagraphBlock.blockName,
-				ParagraphBlock.blockEditorSelector
+				ParagraphBlock.blockEditorSelector,
+				{ noSearch: true }
```

Comment:
The Paragraph block is only directly inserted with the Editor: Post Advanced Flow spec. Since searching for "Paragraph" returns slow loading patterns, the `noSearch` optional param is used to specify that Playwright is to select directly from the block listing that is shown when opening the sidebar.